olive/cli/auto_opt.py

```diff
@@ -90,50 +88,45 @@ def register_subcommand(parser: ArgumentParser):
         add_accelerator_options(sub_parser)
 
         # dataset options
-        dataset_group = sub_parser.add_argument_group(
-            "dataset options, required for some optimization passes like quantization, and evaluation components"
-        )
-        dataset_group.add_argument(
+        sub_parser.add_argument(
             "-d",
             "--data_name",
             type=str,
             help="The dataset name.",
         )
-        dataset_group.add_argument(
+        sub_parser.add_argument(
             "--split",
             type=str,
             help="The dataset split to use for evaluation.",
         )
-        dataset_group.add_argument(
+        sub_parser.add_argument(
             "--subset",
             type=str,
             help="The dataset subset to use for evaluation.",
         )
-        dataset_group.add_argument(
+        sub_parser.add_argument(
             "--input_cols",
             type=str,
             nargs="*",
             help="The input columns to use for evaluation.",
         )
-        dataset_group.add_argument(
+        sub_parser.add_argument(
             "--batch_size",
             type=int,
             default=1,
             help="Batch size for evaluation.",
         )
 
-        auto_opt_config_group = sub_parser.add_argument_group("auto optimizer options")
-        auto_opt_config_group.add_argument(
-            "--precisions",
+        sub_parser.add_argument(
+            "--precision",
             type=str,
-            nargs="*",
```

Comment:
So we only accept one precision for this cli?

Reply:
yes for the auto opt command lline.